### PR TITLE
Progress Tab UI tests

### PR DESCRIPTION
**Pull request by caleybrock**
I removed the angular set of UI tests when removing the angular code. This adds back those tests with some updates to make them work with the new react views.

This really only tests the progress tab, and that you can switch between all tabs on the teacher dashbaord.

TODOs:
- remove `?no_home_redirect=1` everywhere in code. there's no reason to have it now that we have a new dashboard.
- add additional steps/checks for each of the additional tabs

### CODE REVIEW

File: apps/src/templates/sectionProgress/VirtualizedSummaryView.jsx [hunk 1]
```diff
@@ -145,6 +145,7 @@ class VirtualizedSummaryView extends Component {
           styleTopRightGrid={progressStyles.topRight}
           width={styleConstants['content-width']}
           onScroll={onScroll}
+          id={"#uitest-summary-view"}
```
Comment (davidsbailey) [inline apps/src/templates/sectionProgress/VirtualizedSummaryView.jsx]: remove `#`

**Comment by caleybrock** [comment]
This test is passing consistently on ChromeDriver and SauceConnect ChromeLatestWin7, but has failed 3 times in circle. Going to start a separate PR for debug purposes, but since the iteration time is going to be much longer, I'm guessing we'll be without UI tests on teacher dashboard until the end of the week.

**Comment by caleybrock** [comment]
Gotten past loading the dropdown by seeding the express course.

Now timing out on `And I wait until element "#uitest-summary-view" is visible`
I can reproduce this behavior locally by returning a 403 with either the section_level_progress api or the script_structure api. So for some reason the user does not have permission to access the section (which doesn't make a ton of sense) or the script, which could be possible?

**Comment by caleybrock** [comment]
I'm going to remove part of this test and just merge the parts that actually work. Here's what I'm removing:

And I wait until element "#uitest-summary-view" is visible
    And I click selector "#uitest-toggle-detail-view" once I see it
    And I wait until element "#uitest-detail-view" is visible
    And I wait until element "a[href*='/s/allthethings/stage/2/puzzle/1']" is in the DOM
    # Completed bubble is green
    Then element "a[href*='/s/allthethings/stage/2/puzzle/1'] .uitest-bubble" has css property "background-color" equal to "rgb(14, 190, 14)"
    # Not started bubble is gray
    Then element "a[href*='/s/allthethings/stage/2/puzzle/2'] .uitest-bubble" has css property "background-color" equal to "rgb(254, 254, 254)"

**Pull request merged**